app/views/admin/api_docs/base/_form.html.slim

```diff
@@ -17,7 +17,7 @@
   = form.semantic_errors :base_path
   = form.input :skip_swagger_validations, as: :boolean
 = form.buttons do
-  = form.commit_button commit_button_label
+  = form.button commit_button_label, button_html: { class: 'pf-c-button pf-m-primary' } 
```

Comment:
I may be wrong but I think it has to be either `commit_button` or have `type: 'submit'`.

Reply:
Is not needed:
<img width="786" alt="Screenshot 2021-03-04 at 11 47 59" src="https://user-images.githubusercontent.com/13486237/109952813-90927280-7cdf-11eb-9898-d1d42ebbb84a.png">

Reply:
Cool! Didn't know that.